GUI Documentation Changes

msDestiny14 opened the pull request:
https://github.com/BabylonJS/ThePirateCove/issues/108

Still not sure what he means by the last part?
In Saving Locally and Saving To The Snippet Server Link: See Loading GUIs Into The Playground 2

Does he mean line 166 and 176? Not sure how that should be linked instead.

getzelus commented:
Hello, thx for merged it.
When it will be online ? I see that the tutorial page has still this quote code at the bad place.

deltakosh commented:
We merge every monday

getzelus commented:
Hello, I dont know why, there is still this bad display :( 
Can u do something ? 

Here is an example of what's wrong now with the tags : 
http://i.imgur.com/yUSyvZR.jpg

RaananW commented:
Just wanted to update :
it was merged - http://doc.babylonjs.com/tutorials/raycasts

getzelus commented:
Thank you. 
Can you change the last picture for multipick by the good one ? 

https://cloud.githubusercontent.com/assets/24637259/22629048/61651dcc-ebdf-11e6-86e4-4fdea6f16212.jpg

I don't know how to store pictures in the good place so I can't commit for this.

RaananW commented:
Done.
Have an interesting day!

msDestiny14 committed:
documentation fixes

msDestiny14 committed:
documentation fixes

PirateJC committed:
Update toolsAndResourcesGUIEditorLandingPage.md

Fixed anchor links in the save sections to point to the appropriate corresponding load section.

PirateJC commented:
Links have now been fixed to point to the appropriate anchors. 

The specific request was to make sure the links actually work. They do now.
